### PR TITLE
Removed references to ACM from CCS Spec

### PR DESCRIPTION
**Pull request by deboer-tim**

### CODE REVIEW

File: Contest_Control_System_Requirements.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ permalink: /ccs_system_requirements
 
 This document specifies requirements for operation, verification and
 validation of Programming Contest Control Systems that wish to be
-considered for managing the operation of the [ACM International
+considered for managing the operation of the [International
 Collegiate Programming Contest](http://icpc.baylor.edu) [World
```
Comment (niemela) [inline Contest_Control_System_Requirements.md:11]: Link should now be https://icpc.global

**Review by niemela — APPROVED** [review]

**Review by clevengr — APPROVED** [review]

**Comment by deboer-tim** [comment]
If one of the three who has access to merge is the second approver, may as well just merge it at the same time too?

**Comment by eldering** [comment]
> If one of the three who has access to merge is the second approver, may as well just merge it at the same time too?

Indeed. But I was hesitant to merge this since @niemela still had an unanswered comment. That can still be addressed later though...

File: Contest_Control_System_Requirements.md (hunk 1 above)
Comment (deboer-tim) [reply Contest_Control_System_Requirements.md:11]: Sorry, missed this. I will fix via a future PR.

**Pull request merged**